src/plugins/condition/Condition.js

```diff
@@ -37,7 +37,7 @@ import {computeCondition} from "./utils/evaluator";
 *       {
 *           telemetry: '',
 *           operation: '',
-*           input: '',
+*           input: [],
```

Comment:
Please change this to remove the else now:
https://github.com/nasa/openmct/blob/83d08ae3699f9f8bcb6afcdc14879bd8735076f8/src/plugins/condition/criterion/TelemetryCriterion.js#L82

Also, make sure all the tests pass when you do that. Thanks.

Reply:
I had made that change on an earlier branch. Restored it and it works for single or double inputs. Tests pass.